src/signals/IncidentMap/components/IncidentMap.tsx

```diff
@@ -1,8 +1,247 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2022 Gemeente Amsterdam
+import type { Dispatch, FC, SetStateAction } from 'react'
+import { memo, useState, useCallback, useEffect, useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import format from 'date-fns/format'
+import subDays from 'date-fns/addDays'
 
-const IncidentMap = () => {
-  return <>hier kan de kaart komen</>
+import type {
+  Map as MapType,
+  MarkerCluster as MarkerClusterType,
+} from 'leaflet'
+import L from 'leaflet'
+import type { Geometrie } from 'types/incident'
+
+import MAP_OPTIONS from 'shared/services/configuration/map-options'
+import configuration from 'shared/services/configuration/configuration'
+import { featureToCoordinates } from 'shared/services/map-location'
+import { makeSelectFilterParams } from 'signals/incident-management/selectors'
+import useFetch from 'hooks/useFetch'
+import {
+  incidentIcon,
+  markerIcon,
+} from 'shared/services/configuration/map-markers'
+import MarkerCluster from 'components/MarkerCluster/MarkerCluster'
+import DetailPanel from 'signals/incident/components/form/MapSelectors/Asset/Selector/DetailPanel/DetailPanel'
+
+import type { IncidentSummary } from 'components/OverviewMap/types'
+import styled from 'styled-components'
+import { breakpoint } from '@amsterdam/asc-ui'
+import { AssetSelectProvider } from 'signals/incident/components/form/MapSelectors/Asset/context'
+import type { Location } from 'types/incident'
+import Map from '../../../components/Map'
+import type { AssetSelectValue } from '../../incident/components/form/MapSelectors/Asset/types'
+
+export const Wrapper = styled.div`
+  position: relative;
+  height: 100%;
+  width: 100%;
+  box-sizing: border-box; // Override box-sizing: content-box set by Leaflet
+  z-index: 2; // position over the site header
+  display: flex;
+
+  @media screen and ${breakpoint('max-width', 'tabletM')} {
+    flex-direction: column;
+  }
+`
+
+export const StyledMap = styled(Map)`
```

Comment:
Map is een implementatie van van styled components, met incidentContainer state erin. Dit heb je niet nodig.

Reply:
op te lossen door closeMap() en mapactive door te geven als props of via context en optioneel te maken